src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handlerKeyUp);
+  }
+
+  componentDidUpdate() {
+    document.removeEventListener('keyup', this.handlerKeyUp);
+  }
+
+  handlerKeyUp = (event: KeyboardEvent) => {
+    this.setState({ message: `The last pressed key is [${event.key}]` });
```

Comment:
It's better to save only the `event.key` value in the state because it minimizes storage requirements and simplifies state management.